### PR TITLE
Add support for the new users by email API

### PR DESCRIPTION
**Pull request by erichard**
https://support.auth0.com/notifications/59eff54453e4cf000ab54469

### CODE REVIEW

File: src/API/Management/UsersByEmail.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace Auth0\SDK\API\Management;
+
+class UsersByEmail extends GenericResource
+{
+    public function get($params = array())
+    {
+        $client = $this->apiClient->get()
+            ->addPath('users-by-email')
+        ;
```
Comment (cocojoe) [inline src/API/Management/UsersByEmail.php]: This semi-colon looks lonely.

**Comment by erichard** [comment]
@cocojoe I've fixed the CS

**Comment by cocojoe** [comment]
☝️ @glena

**Comment by glena** [comment]
👍

**Comment by erichard** [comment]
@glena @cocojoe Could you tag a stable version with this PR please ?

**Comment by cocojoe** [comment]
Sure, I wanted to wait for something else but it's probably a minor so I'll do a patch for this.

**Pull request merged**